src/components/LiteracyComponents/ActivitySettingModal.tsx

```diff
@@ -100,7 +100,7 @@ function ActivitySettingModal(props: Props): React.ReactElement {
             </Grid>
             <Grid item style={{width: '100%', paddingTop: '0.5em'}}>
               <Typography style={{fontFamily: 'Arimo'}}>
-                Select or create a label describing the type of {checklistType === 'ReadingTeacher' ? 'text read during this observation' : 'activity that was happening for most of this observation'}:
+                Select {checklistType === 'ReadingTeacher' ? 'or create' : null} a label describing the type of {checklistType === 'ReadingTeacher' ? 'text read during this observation' : 'activity that was happening for most of this observation'}:
```

Comment:
In case of falsy `checklistType` a double space character will be rendered
```
Select {null} a label
```
is converted to
```
Select  a label
```
Let's fix that by adding a space somewhere in the ternary condition, for instance
```suggestion
                Select {checklistType === 'ReadingTeacher' ? 'or create ' : ' '}a label describing the type of {checklistType === 'ReadingTeacher' ? 'text read during this observation' : 'activity that was happening for most of this observation'}:
```